tsnet/tsnet.go

```diff
@@ -323,6 +323,24 @@ func (s *Server) doInit() {
 	}
 }
 
+// TailscaleIPs returns IPv4 and IPv6 addresses for this node. If the node
+// has not yet joined a tailnet or is otherwise unaware of its own IP addrsses,
+// netip.IsUnspecified() addresses will be returned.
```

Comment:
Unspecified is for `0.0.0.0` or `::0`

You want `!IsValid`

https://pkg.go.dev/net/netip#Addr.IsUnspecified

Reply:
Updated description.